### ISSUE TITLE
Lower peak area calculation than Skyline and Qual Browser

### ISSUE THREAD

**Issue by jamesjiadazhan** [mzmine/mzmine#1900]
### Basic information

* My operating system and version: Windows 11
* My mzmine version: MZMINE 4

### What happened

* Please describe what happened and how it differs from what you expected to happen.
I was trying to compare my untargeted feature extraction from MZMINE with the targeted feature extraction from Skyline and Thermo Fisher Qual Browser. It is actually a surprise to find that their peak area calculation can be largely different even for the same feature.

For example, the peak area from MZMINE for m/z 86.09645 is 6E6. The result is the same by checking with the chromatogram built from the ADAP chromatogram builder.

<img width="689" alt="image" src="https://github.com/mzmine/mzmine/assets/108076575/34cfd573-dfaa-4146-8124-f5b495054f2f">
<img width="573" alt="image" src="https://github.com/mzmine/mzmine/assets/108076575/6783fb8e-0d12-4b6b-83f9-3fb35a9a7c85">

The peak area from Skyline for the same feature is: 1.4E9.
<img width="553" alt="image" src="https://github.com/mzmine/mzmine/assets/108076575/ed20e7bd-79c7-444b-83cf-ed95fc6473b4">

The peak area from Qual Browser for the same feature is:  4.4E7.
<img width="648" alt="image" src="https://github.com/mzmine/mzmine/assets/108076575/802f4381-ca26-440b-b4a7-758cbd6537e9">

* Please include all parameter values for the mzmine modules you used. 
- Mass detection
  - noise level 1E4
- ADAP chromatogram builder
  - Min group # scan: 5
  - Group intensity threshold: 3E4
  - Min highest intensity: 7E4
  - Scan to scan accuracy: 0 m/z or 5 ppm

* **Please upload a standalone mzmine project file (or a batch + raw files) that demonstrates the problem you are reporting.** You can use Google Drive, Dropbox, or a similar repository to upload the project.
- Here is the example data to replicate the issue: https://drive.google.com/drive/folders/1LlIR9PdThlyh9O5Q5QRIcZq7My7-OwRz?usp=sharing

Thank you so much for your help!

**Comment by jamesjiadazhan** [comment]
I think I just found the answer to why the peak area calculation is different. To sum up, it is because MZMINE uses centroid mzXML data. I used centroid data in MZMINE because it is much faster and it typically freezes if I don't do so when therea are hundreds of files. Below are the Skyline answers:
- https://skyline.ms/announcements/home/support/thread.view?rowId=37925
_In the attached picture you can see that Skyline summed nine profile points between 674.5119 and 674.5253. That's how Skyline came up with the total sum of 19.3 million._
- ![image](https://github.com/mzmine/mzmine/assets/108076575/4b953401-8ff0-4f30-a032-a94eec354cfe)

_If you change the mass analyzer to "Centroid" then Skyline will use Thermo's centroiding algorithm so that there is only one point in the spectrum, and that point ends up with the intensity 4.99 million, which is the same as you see in XCalibur._



I did a little search on the profile and centroid algorithm difference:

Profile Algorithm
- Profile Data: In LC-MS, profile data represent the raw, continuous signal measured by the detector. This signal is typically high-resolution and includes all the fine details of the peak shape.
- Peak Area Calculation: The area under the curve (AUC) for profile data is calculated by integrating the entire signal over the range of m/z values that correspond to the peak. This includes all the small fluctuations and noise that are part of the raw signal.
- 
Centroiding Algorithm
- Centroid Data: Centroiding simplifies the profile data by reducing it to discrete points that represent the centers (centroids) of the detected peaks. This process involves identifying the peak maxima and assigning a single m/z value and intensity to each peak.
- Peak Area Calculation: The centroided data consist of these discrete points, and the peak area is typically calculated based on the summed intensities of these centroids. This method may exclude some of the lower intensity points and noise that are present in the profile data.

**Comment by SteffenHeu** [comment]
To add to that: the rest of the difference is probably down to mzmine using minutes rather than seconds for the area calculation (factor of 60) and different integration widths in mz or RT range.

**Comment by jamesjiadazhan** [comment]
> To add to that: the rest of the difference is probably down to mzmine using minutes rather than seconds for the area calculation (factor of 60) and different integration widths in mz or RT range.

Thanks for your feedback!

**Comment by symbolic89** [comment]
I am not sure if I can continue in this issue. If not, I will open a new one.  :) 

In our lab my colleague and I have encountered the same problem as described here, where peak areas in Xcalibur are roughly 60 times higher than in MZmine 4 and the reason for this I have found probably in this quote:

> To add to that: the rest of the difference is probably down to mzmine using minutes rather than seconds for the area calculation (factor of 60) and different integration widths in mz or RT range.

My colleague worked until now with an older version of MZmine (2.5.3) and we are trying with a dataset to compare the results in MZmine 2.5.3 with the results of MZmine 4. The old version gave us the same peak areas as in Xcalibur. In our institute we also have MZmine  3 installed and also this version gives us that difference of approximately factor 60. Am I right to assume that the peak areas are now calculated differently in MZmine than in the old version 2.5.3? And if so, can we change this area calculation in MZmine 4 so that it is easier for us to compare the data sets. 

Thanks in advance for the answers and feedback!
